UPGRADE-14.0.md

```diff
@@ -190,4 +190,8 @@ Follow the instructions in relevant sections, e.g. `shopsys/coding-standards` or
 -   fix Comparison for not logged in users ([#2905](https://github.com/shopsys/shopsys/pull/2905))
     -   unified code for Comparison and Wishlist
     -   refactored Zustand store to use only one store (User Store) for all cartUuid, wishlistUuid and comparisonUuid
+-   added skeletons for Wishlist, Comparison and My orders, Order detail and Product Variants page ([#2906](https://github.com/shopsys/shopsys/pull/2906))
+    -   see #project-base-diff to update your project
+-   fix Add to cart/wishlist/comparison feature when using multiple tabs ([#2906](https://github.com/shopsys/shopsys/pull/2906))
+    -   As fresh unlogged user without `cartUuid` I open multiple tabs, then in each tab I add product to cart, I am experiencing that each tab contains different products in cart, even after refresh. Same applies to `Wishlist` and `Comparison`. This was solved by adding `BroadcastChannel` middleware to Zustand store, so now it uses always only first returned `uuid`.
```

Comment:
I think there are several more things we should add here:

- there is no redirect to homepage after logout
- useRemoveProductFromCart changed its API, this is a breaking change and can break people's apps (here, I would still prefer to not change the API as described below)
- the new broadcast channel can possibly have problems with some more complex data structures. This can potentially break people's apps and they should know about it
- I would mention upgrade of a package (zustand)
- we now rehydrate zustand on the client, so we should tell them that they do not have to check for the initial state in their apps (you removed it for wishlist and comparison, so you can mention it as an example to follow)
- and the last point, which isnt super strong, but still worth consideration, I would mention that there are new skeleton types. It is possible that they have already implemented their own or dont have some of the pages. In such case, they should make an informed decision about it

Reply:
Yup, thanks for help with summary. Just few notes.

I won't include warning about BroadcastChannel, as I've answered with previous comment - I don't see any reason why it would break. And about additional skeletons there is another note in the root tree.